cmd/query/app/server.go

```diff
@@ -81,29 +84,27 @@ func NewServer(logger *zap.Logger, healthCheck *healthcheck.HealthCheck, querySv
 		return nil, errors.New("server with TLS enabled can not use same host ports for gRPC and HTTP.  Use dedicated HTTP and gRPC host ports instead")
 	}
 
-	grpcServer, err := createGRPCServer(querySvc, metricsQuerySvc, options, tm, logger, tracer)
+	grpcServer, err := createGRPCServer(querySvc, metricsQuerySvc, options, tm, telset.Logger, telset.TracerProvider)
```

Comment:
since these are local functions, you can change them now to pass telset. Let's minimize the need to come back to certain files.